src/datasources.js

```diff
@@ -281,14 +286,18 @@ define([
                             info.hashtag = '#case/' + ref.subset;
                         }
                     }
-                    path = "instance('" + source.id + "')" + source.path +
-                           "[" + ref.key + " = " + path + "]";
+                    var keyPath = path;
+                    path = "instance('" + source.id + "')" + source.path;
+                    if (ref.subset_filter && ref.subset_key && ref.subset) {
```

Comment:
I think it could make sense if we ever needed it, but I'm reluctant to do that now since we don't.